### PR TITLE
Add interactive chart where you can vote on my coolness.

### PR DESCRIPTION
**Pull request by Sami-2000**
Add a new servlet to deal with responses, a new form to submit votes, and a new chart to display results.

### CODE REVIEW

File: portfolio/src/main/webapp/charts.js [hunk 1]
```diff
@@ -5,6 +5,30 @@ google.charts.setOnLoadCallback(drawCharts);
 function drawCharts() {
   drawStaticPieChart();
   drawLineChart();
+  drawBarChart();
+}
+
+/*Creates interactive bar chart saying if I'm cool.*/
```
Comment (billyeh) [inline portfolio/src/main/webapp/charts.js]:
```suggestion
/* Creates interactive bar chart saying if I'm cool. */
```

File: portfolio/src/main/webapp/charts.js [hunk 2]
```diff
@@ -5,6 +5,30 @@ google.charts.setOnLoadCallback(drawCharts);
...
+    data.addColumn('string', 'Cool?');
+    data.addColumn('number', 'Votes');
+    Object.keys(coolVotes).forEach((coolCategory) => {
+      data.addRow([coolCategory, coolVotes[coolCategory]]);
+    });
+
+    const options = {
+      'title': 'Am I Cool?',
+      'width':800,
+      'height':1200
```
Comment (billyeh) [inline portfolio/src/main/webapp/charts.js]:
```suggestion
      'width': 800,
      'height': 1200,
```

File: portfolio/src/main/webapp/charts.js (hunk 1 above)
Comment (Sami-2000) [reply portfolio/src/main/webapp/charts.js]: Thank you!

File: portfolio/src/main/webapp/charts.js [hunk 3]
```diff
@@ -5,6 +5,30 @@ google.charts.setOnLoadCallback(drawCharts);
 function drawCharts() {
   drawStaticPieChart();
   drawLineChart();
+  drawBarChart();
+}
+
+/*Creates interactive bar chart saying if I'm cool.*/
+function drawBarChart() {
+  fetch('/cool-data').then(response => response.json())
+  .then((coolVotes) => {
```
Comment (billyeh) [inline portfolio/src/main/webapp/charts.js]: nit: looks like the .then() should be indented by 4 spaces? 
https://google.github.io/styleguide/jsguide.html#formatting-function-expressions

File: portfolio/src/main/java/com/google/sps/servlets/CoolDataServlet.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/cool-data")
+public class CoolDataServlet extends HttpServlet {
+
```
Comment (billyeh) [inline portfolio/src/main/java/com/google/sps/servlets/CoolDataServlet.java:17]: Short documentation comment for this hashmap would be great.

**Review by billyeh — APPROVED** [review]
nice!

File: portfolio/src/main/webapp/charts.js (hunk 3 above)
Comment (Sami-2000) [reply portfolio/src/main/webapp/charts.js]: Thanks for the catch!

**Pull request merged**